Scripts/Python/nb01RPSGame.py

```diff
@@ -710,6 +732,15 @@ def _SendLocalStatusMsg(self, msg):
         """Sends a status chat message (purple text) to the local player's KI."""
         PtSendKIMessage(kKILocalChatStatusMsg, msg)
 
+    def _QuitGame(self, YesNo):
+        """Performs the standup animation when sitting down
+           Disable control key events
+           Enables the disabled movement keys
+           PtAvatarExitAFK() is actually just performing a GoToStage 2 which for the sit brain is the standup animation - https://github.com/H-uru/Plasma/blob/9956967363f383d91f43162d116ad3477148adb2/Sources/Plasma/FeatureLib/pfPython/cyAvatar.cpp#L1970"""
```

Comment:
This should probably just list the C++ function that performs the GoToStage. We already have the source code in front of us, so a URL is a bit overkill.